### ISSUE TITLE
Fix section links styling for accessibility compliance

### ISSUE THREAD

**Issue by AnastasiaLd** [dbnschools/moodle-theme_fordson#130]
Hi,

The styling applied to some of the section hyperlinks is not compliant with the current W3C's Web Content Accessibility Guidelines – WCAG2.  Ideally, the Boost theme accessible styling for links should be preserved.

Additionally, we noticed some inconsistency in the link styling format.

```
section#region-main p a {
    color: #14a9fe;
    text-decoration: underline;
    font-weight: 700;
}

a {
    color: #1c1c1c;
    text-decoration: none;
    background-color: transparent;
}

.format-tiles .embed_cm_modal a {
    color: #979797;
}

```
<img width="339" alt="links_example_fordson" src="https://user-images.githubusercontent.com/32487197/106478165-da015f00-64a0-11eb-92dc-762e50447cf2.PNG">
<img width="462" alt="link_embeded_example_fordson" src="https://user-images.githubusercontent.com/32487197/106478182-dc63b900-64a0-11eb-9958-09847adf1004.PNG">
<img width="475" alt="link_example_boost" src="https://user-images.githubusercontent.com/32487197/106478862-93603480-64a1-11eb-85e4-084061fe9e6b.PNG">
